articles/client-auth/v2/mobile-desktop.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Authentication for Mobile & Desktop Apps
+description: Explains how to authenticate users in a mobile or desktop application.
+toc: true
+---
+
+# Authentication for Mobile & Desktop Apps
+
+You can authenticate users of your mobile/desktop applications by:
+
+* Using the [Lock](/libraries/lock) client libraries;
+* Calling the Auth0 [Authentication API](/api/authentication) endpoints.
+
+This article will cover how to call the Auth0 [Authentication API](/api/authentication) endpoints using [Proof Key for Code Exchange (PKCE)](/api-auth/grant/authorization-code-pkce) during the authentication process.
```

Comment:
Maybe link to the alternative? Something like: `If you are interested in implementing this with Lock, refer to [Lock v2 for iOS](/libraries/lock-ios) or [Lock for Android](/libraries/lock-android)`.